{
	"id": "CVE-2019-5010",
	"tags": [
		{
			"label": "Product",
			"message": "Python"
		},
		{
			"label": "Version",
			"message": "n/a"
		},
		{
			"label": "Vulnerability",
			"message": "CWE-476: NULL Pointer Dereference"
		}
	],
	"description": " An exploitable denial-of-service vulnerability exists in the X509 certificate parser of Python.org Python 2.7.11 / 3.6.6. A specially crafted X509 certificate can cause a NULL pointer dereference, resulting in a denial of service. An attacker can initiate or accept TLS connections using crafted certificates to trigger this vulnerability.",
	"poc": {
		"references": [
			"https://talosintelligence.com/vulnerability_reports/TALOS-2019-0758"
		],
		"githubs": [
			"https://github.com/0xT11/CVE-POC",
			"https://github.com/JonathanWilbur/CVE-2019-5010",
			"https://github.com/developer3000S/PoC-in-GitHub",
			"https://github.com/hectorgie/PoC-in-GitHub",
			"https://github.com/nomi-sec/PoC-in-GitHub"
		]
	}
}
